packages/tracing/src/browser/browsertracing.ts

```diff
@@ -155,3 +229,15 @@ export function getMetaContent(metaName: string): string | null {
   const el = document.querySelector(`meta[name=${metaName}]`);
   return el ? el.getAttribute('content') : null;
 }
+
+/** Adjusts transaction value based on max transaction duration */
+function adjustTransactionDuration(maxDuration: number): BeforeFinishCallback {
+  return (transaction: IdleTransaction, endTimestamp: number): void => {
+    const diff = endTimestamp - transaction.startTimestamp;
+    const isOutdatedTransaction = endTimestamp && (diff > maxDuration || diff < 0);
+    if (isOutdatedTransaction) {
+      transaction.setStatus(SpanStatus.Cancelled);
```

Comment:
Should it be `SpanStatus.DeadlineExceeded` instead?

Reply:
Yes it should, thanks.